modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -607,7 +607,9 @@ public interface Preferences extends ObservableMap<String, Object> {
         Color getForegroundColor();
 
         /**
-         * The accent color.
+         * The accent color, which is usually a vivid color that contrasts with the foreground
+         * and background colors. It can be used to highlight the active or important part of a
+         * control and make it stand out from the rest of the user interface.
```

Comment:
I think it's more important to say what it's used for than what value is should have, so I would switch the order:
```
The accent color, which can be used to highlight the active or important part of a
control and make it stand out from the rest of the user interface. It is usually a vivid
color that contrasts with the foreground and background colors.
```

Reply:
Yes, that's better.